stubs/Pillow/PIL/ImageQt.pyi

```diff
@@ -3,20 +3,20 @@ from typing_extensions import Literal, TypeAlias
 
 from .Image import Image
 
-QImage: TypeAlias = Any  # imported from either of {PyQt6,PySide6,PyQt5,PySide2}.QtGui
-QPixmap: TypeAlias = Any
+_QImage: TypeAlias = Any  # imported from either of {PyQt6,PySide6,PyQt5,PySide2}.QtGui
```

Comment:
Won't this change cause type checkers to emit false-positive errors if users do `from PIL.ImageQT import QImage`?

Reply:
Doesn't feel like it's meant to be re-exported. You'd do `from PyQt6.QtGui import QImage`. It also only exists (because it's only imported) if the corresponding Qt bindings library is installed.
Also not referenced in the doc/API: https://pillow.readthedocs.io/en/stable/reference/ImageQt.html
(even most other public methods are wrapped in `PIL.Image`)

Reply:
That makes sense, thank you!